src/StripeTests/Entities/CountrySpecTest.cs

```diff
@@ -0,0 +1,24 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+    public class CountrySpecTest : BaseStripeTest
+    {
+        [Fact]
+        public void Deserialize()
+        {
+            string json = GetFixture("/v1/country_specs/US");
+            var countrySpec = Mapper<CountrySpec>.MapFromJson(json);
+            Assert.NotNull(countrySpec);
+            Assert.IsType<CountrySpec>(countrySpec);
+            Assert.NotNull(countrySpec.Id);
+            Assert.Equal("country_spec", countrySpec.Object);
```

Comment:
Part of me still wants to test deserialization of complex sub-resource types but I think we want to avoid this and just trust the framework.

Reply:
Yeah, I think we'd only need in-depth tests for cases where we have custom deserializers. (Thankfully that doesn't seem to be the case in stripe-dotnet thanks to API version pinning.) In "simple" cases like this, we'd only be testing the Newtonsoft.Json library rather than stripe-dotnet.

Reply:
Yeah so not entirely. We'd also be testing the proper nesting and naming of properties too for example which is easy to mess up. But yes I get your point, I just could not resist raising it one last time :p